src/GitHub.VisualStudio/UI/Views/Controls/RepositoryCloneControl.xaml

```diff
@@ -137,7 +137,7 @@
           <Setter Property="Foreground"
                   Value="{DynamicResource GHTextBrush}" />
           <Setter Property="Margin"
-                  Value="0,6,12,6" />
+                  Value="0,0,12,0" />
```

Comment:
So maybe we want to keep the padding here, as it's not responsible for the clipping after all.